daffodil-cli/src/main/scala/org/apache/daffodil/InfosetTypes.scala

```diff
@@ -108,22 +108,22 @@ object InfosetType extends Enumeration {
     forPerformance: Boolean): InfosetHandler = {
 
     infosetType match {
-      case InfosetType.EXI => EXIInfosetHandler(dataProcessor)
-      case InfosetType.EXISA => EXIInfosetHandler(dataProcessor, schemaUri.get)
-      case InfosetType.JDOM => JDOMInfosetHandler(dataProcessor)
-      case InfosetType.JSON => JsonInfosetHandler(dataProcessor)
-      case InfosetType.NULL => NULLInfosetHandler(dataProcessor)
-      case InfosetType.SAX => SAXInfosetHandler(dataProcessor, forPerformance)
+      case InfosetType.EXI       => EXIInfosetHandler(dataProcessor)
+      case InfosetType.EXISA     => EXIInfosetHandler(dataProcessor, schemaUri.get)
+      case InfosetType.JDOM      => JDOMInfosetHandler(dataProcessor)
+      case InfosetType.JSON      => JsonInfosetHandler(dataProcessor)
+      case InfosetType.NULL      => NULLInfosetHandler(dataProcessor)
+      case InfosetType.SAX       => SAXInfosetHandler(dataProcessor, forPerformance)
       case InfosetType.SCALA_XML => ScalaXMLInfosetHandler(dataProcessor)
-      case InfosetType.W3CDOM => W3CDOMInfosetHandler(dataProcessor)
-      case InfosetType.XML => XMLTextInfosetHandler(dataProcessor)
+      case InfosetType.W3CDOM    => W3CDOMInfosetHandler(dataProcessor)
+      case InfosetType.XML       => XMLTextInfosetHandler(dataProcessor)
```

Comment:
Are there an opinions on this kind of alignment? I tend to not like becaause if we add a new longer case we have to reindent the whole thing and it makes the diff harder to see what actually changed.

Reply:
I personally like it. gofmt does the same with struct fields and makes everything look nice. You can omit whitespace changes in a github diff by clicking on the gear icon drop down, selecting "hide whitespace", and then clicking on apply.

Reply:
Yeah, I guess github does show clearly what is whitespace. Seems reasonable to me :+1:

Reply:
I profoundly hate this aligning the arrows crap. 
Only looks good on toy situations.

E.g., something more real like:

```
foo match {
case S(bar, baz) if (f(baz) == g(bar)) => // ok
case T(bar)                                           => bar 
case _                                                   => Assert.invariantFailed("now I'm completely out of line space to put this message.")
}
```
As soon as some condition is slightly more complex than the others, the arrows all end up way off to the right and it is a mess.

Reply:
I've disabled all alignments with `align.preset = none`